components/list/list-item-checkbox-mixin.js

```diff
@@ -7,28 +7,36 @@ import { nothing } from 'lit-html';
 import { SelectionInfo } from '../selection/selection-mixin.js';
 import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
 
+/**
+ * @property label - The hidden label for the checkbox if selectable
+ */
 export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(LabelledMixin(superclass)) {
 
 	static get properties() {
 		return {
 			/**
-			 * Disables the input
+			 * **Selection:** Disables the input
```

Comment:
I added these for selection and drag & drop so that it's easier to identify properties for a specific state in the giant properties table. I also was thinking we could sort by that (separate PR in documentation to come for that)